src/components/views/rooms/CollapsibleButton.tsx

```diff
@@ -25,18 +25,25 @@ interface ICollapsibleButtonProps extends ComponentProps<typeof MenuItem> {
     title: string;
 }
 
-export const CollapsibleButton = ({ title, className, ...props }: ICollapsibleButtonProps) => {
+export const CollapsibleButton = ({ title, children, className, ...props }: ICollapsibleButtonProps) => {
     const inOverflowMenu = !!useContext(OverflowMenuContext);
     if (inOverflowMenu) {
         return <MenuItem
             {...props}
             className={classNames("mx_CallContextMenu_item", className)}
         >
             { title }
+            { children }
```

Comment:
is anywhere using children anymore? also was this even needed? `{...props}` can pass children too.

Reply:
No, but I figure if we take `children` in our props we should render it.  If you disagree, happy to revert.